src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -112,7 +117,46 @@ public void testVCFGZIndex_tabix() {
         checkIndex(index, Arrays.asList("1", "2", "3", "4"));
     }
 
-    @Test(enabled=false)
+    @Test
+    public void testVCFGZLargeHeaderIndex_tabix() throws IOException {
+        // copy the input file, and create an index
+        final File inputVCF = getTestFile("4featuresHG38Header.vcf.gz");
+        final File tempDir = createTempDir("testVCFGZLargeHeaderIndex");
+        final File inputCopy = new File(tempDir, inputVCF.getName());
+        Files.copy(inputVCF.toPath(), inputCopy.toPath());
+        final File outIndexFile = new File(tempDir, inputCopy.getName() + TabixUtils.STANDARD_INDEX_EXTENSION);
+
+        final String[] args = {
+                "--feature_file" ,  inputCopy.getAbsolutePath(),
+                "-O" ,  outIndexFile.getAbsolutePath()
+        };
+        final Object res = this.runCommandLine(args);
+        Assert.assertEquals(res, outIndexFile.getAbsolutePath());
+
+        // use the location of every variant in the input as a query interval for the indexed copy of the same file
+        try (final FeatureDataSource<VariantContext> originalSource = new FeatureDataSource<>(inputVCF);
+                final FeatureDataSource<VariantContext> indexedSource = new FeatureDataSource<>(inputCopy)) {
+            Iterator<VariantContext> originalIterator = originalSource.iterator();
+            while (originalIterator.hasNext()) {
+                final VariantContext originalVC = originalIterator.next();
+                final Iterator<VariantContext> indexedIterator = indexedSource.query(
+                        new SimpleInterval(
```

Comment:
there's a constructor directly from another locatable if you want to save a few characters